N/nativefiledialog_extended/build_tarballs.jl

```diff
@@ -0,0 +1,102 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "nativefiledialog_extended"
+version = v"1.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/btzy/nativefiledialog-extended.git", "5786fabceeaee4d892f3c7a16b243796244cdddc")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/nativefiledialog-extended
+# NATIVE  mingw, apple
+# PORTAL  linux ~armv6l
+# GTK3    linux ~armv6l, freebsd
+# SDL2    linux  armv6l
+CMAKE_FLAGS="-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_CXX_STANDARD=11 \
+    -DCMAKE_C_FLAGS=\"-Wall -Wextra -pedantic\" \
+    -DCMAKE_CXX_FLAGS=\"-Wall -Wextra -pedantic\" \
+    -DBUILD_SHARED_LIBS=ON"
+
+# NATIVE and PORTAL
+BUILD="build"
+PREFIX="${prefix}"
```

Comment:
What's the point of setting a variable to another variable with the same name?

Reply:
Bash variables are case-sensitive (I think), making `prefix` and `PREFIX` different. The point was to make it easier to read only the "configs" (`PREFIX`, `BUILD`, `NFD_PORTAL`, `NFD_APPEND_EXTENSION`) before every `cmake` call.